tweak debug.bootTime event spec, add test and docs, bump version (#70)
Actyx/machines: machine-runner/README.md

})
```

#### Event List

##### `next`

A `next` event is emitted when a state transition happens and the machine runner has processed all events matching the supplied tag.

The payload is `StateOpaque`, similar to the value produced in the `for-await` loop. 

##### `error`

An `error` event is emitted when an error happened inside the runner. Currently this is the list of the errors:

> - A command is called when locked i.e. another command is being issued in the same machine

Reviewer: Perhaps in addition to the problem statement, explain that after invoking a command the app needs to await the resulting Promise to see whether it was successful (in which case it needs to await the next state update) or unsuccessful (in which case it can try again).

Author: This explanation should go to another section. Yet, this part of our API is not mature yet.
Currently, a failing command does not throw.
In this version, a failing command returns `Promise<void>`, while a successful one returns `Promise<Metadata>`.
I'll open an issue to decide about this and put this on version 0.4.6

Author: Will address this in https://github.com/Actyx/machines/issues/71

- A command is called when the corresponding state is expired i.e. another command has been successfully issued from that state
- A command is called on a destroyed machine

The payload has an error subtype.

##### `change`

A `change` event is emitted when a `next` event is emitted, a command is issued, a command is published, or subscription error happened due to losing a connection to Actyx. This event is particularly useful in UI code where not only state changes are tracked, but also command availability and errors.

> The payload is `StateOpaque`, similar to the value produced in the `for-await` loop. 

Reviewer: ```suggestion
The payload is of type `StateOpaque`, like the value produced in the `for-await` loop. 
```

##### `debug.bootTime`

> A `debug.bootTime` event is emitted when a machine runner have caught up with an Actyx subscriptions (i.e. finished processing its events until the latest one) for the first time.

Reviewer: ```suggestion
A `debug.bootTime` event is emitted when a machine runner has caught up with its Actyx subscription (i.e. finished processing its events until the latest one) for the first time.
```
